EU4toV2/Source/Mappers/CultureGroups/CultureGroup.h

```diff
@@ -18,6 +18,7 @@ class CultureGroup: commonItems::parser
 
 	void addNeoCulture(const std::string& theName, const std::shared_ptr<Culture>& culture, const std::string& oldCulture);
 	void mergeCulture(const std::string& theName, const std::shared_ptr<Culture>& culture);
+	void setUnionTag(const std::string& tag) { culturalUnionTag = tag; }
```

Comment:
You're only using this to clear the tag, so replace with
```
void clearUnionTag() { cultureUnionTag.clear; }
```

Reply:
I foresee setUnionTag being more useful in the future than clearUnionTag

Reply:
we'll add set when it's actually needed, don't worry about it. Having clear naming for the purpose it's used for is more important.